1_The_Physical/Guess_Point_Level_1_Stateless/2_Arrange_Act_Assert/2_2_Stats_Calculator/src/index.spec.ts

```diff
@@ -26,73 +26,62 @@ describe('stats calculator', () => {
   });
 
   describe('Calculate minimum value in a sequence', () => {
```

Comment:
Great commit here refactoring to parameterization 🔥 your stuff is looking super clean.